webapp/components/experiments/ExperimentEdit.js

```diff
@@ -360,7 +361,7 @@ const ParameterRow = (parameter, index, handleParamSelection, handleChange, hand
                 id={`${parameter.name}-values`}
                 label="Values (separated with comas)"
                 variant="filled"
-                value={parameter?.val || ''}
+                value={parameter?.val || parameter?.values.join('')}
```

Comment:
should be joined by ",". Currently this converts [1,2,3,4] to '1234'

Reply:
@lrebscher  The `values` in the list currently returns an array like this - values: (7) ["9", ",", "0", ",", "8", ",", "7"], that's the reason using this parameter?.values.join(''), which gives "9,0,8,7"